ks3-jdk/src/jvmMain/kotlin/io/ks3/java/time/YearMonthAsIntSerializer.kt

```diff
@@ -0,0 +1,18 @@
+package io.ks3.java.time
+
+import io.ks3.java.intSerializer
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.Serializable
+import java.time.YearMonth
+import kotlin.math.abs
+
+typealias YearMonthAsInt = @Serializable(with = YearMonthAsIntSerializer::class) YearMonth
+
+object YearMonthAsIntSerializer : KSerializer<YearMonth> by intSerializer(
```

Comment:
This one needs a little KDoc to explain the purpose, because it took me a second to figure out what it does. So it will take March 2020 and encode it as 202003? That seems pretty odd, I would have thought just encoding it as a string "2020-03" would be a lot more common.

Reply:
Yeah, I agree. I'll just delete it instead. I doubt it'll be used actually.